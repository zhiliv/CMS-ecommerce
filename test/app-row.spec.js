import { shallowMount } from '@vue/test-utils'
import Row from '@/components/app/row/row.vue'

describe('app-row.js', () => {
  let cmp

  beforeEach(() => {
    cmp = shallowMount(Row, {
      // Создать поверхностный экземпляр компонента
      propsData: {
        rowCols: 'auto',
        rowColsSm: 'auto',
        rowColsMd: 'auto',
        rowColsLg: 'auto',
        rowColsXl: 'auto',
        rowColsXxl: 'auto',
        classes: 'simple',
        styles: 'background: white',
        id: 'indent'
      },
    })
  })
  it('Содержимое html для app-row ', () => {
    expect(cmp.element).toMatchSnapshot()
  })
})
