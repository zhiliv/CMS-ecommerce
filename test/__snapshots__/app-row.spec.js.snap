// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`app-row.js Содержимое html для app-row  1`] = `
<div
  class="row row-cols-auto row-cols-sm-auto row-cols-md-auto row-cols-lg-auto row-cols-xl-auto row-cols-xxl-auto simple"
  id="indent"
  style="background: white;"
/>
`;
